CharlieBackend.AdminPanel/Services/CourseService.cs

```diff
@@ -25,9 +20,21 @@ public CourseService(IApiUtil apiUtil, IMapper mapper)
             _mapper = mapper;
         }
 
+        public async Task<CourseDto> DisableCourseAsync(long id)
```

Comment:
Please update return type on disable course action so we won't get internal server error (HTTP 500)

Reply:
Solved